test/issue-466-test.js

```diff
@@ -0,0 +1,144 @@
+"use strict";
+
+const { FakeTimers, assert, refute } = require("./helpers/setup-tests");
+const { sinon } = require("@sinonjs/referee-sinon");
+var timersModule;
+
+if (typeof require === "function" && typeof module === "object") {
+    try {
+        timersModule = require("timers");
+    } catch (e) {
+        // ignored
+    }
+}
```

Comment:
Just use Mocha's built-ins to selectively run it in supported environments
```javascript
        before(function () {
            if (!timersModule) {
                this.skip();
            }
        });
```
Enables a bit better reporting :)